UserInterface/SwitchDemos/SwitchDemos/SwitchDemos/SwitchXamlPage.xaml

```diff
@@ -0,0 +1,24 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:d="http://xamarin.com/schemas/2014/forms/design"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+             mc:Ignorable="d"
+             x:Class="SwitchDemos.SwitchXamlPage"
+             Title="Switch XAML Demo">
+    <ContentPage.Content>
+        <StackLayout>
+            <Label x:Name="SwitchStateLabel"
```

Comment:
Our convention is to start names with a lower case char (as you've done in the code version).